docs/assets/img/exportword.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20"><path d="M13 0l5 4.5V20H2V0h11zm-1.5 1.5h-8v17h13v-12h-5v-5zM15 15v1H5v-1h10zm0-3v1H5v-1h10zm0-3v1H5V9h10zm-5-3v1H5V6h5zm3-4v3h3.3L13 2zm-3 1v1H5V3h5z"/></svg>
```

Comment:
The name of this file could reflect its content, i.e. that it represents a book icon.